### PR TITLE
Sorr task568 convert contributor scoring gdoc to markdown

### PR DESCRIPTION
**Pull request by why2000**
#568 
Also, modified the docker running command to make sure the generated file is still under the current user:group (instead of root:root).

### CODE REVIEW

**Review by DanilYachmenev — COMMENTED** [review]
actually, almost impeccable
and nice notice here https://github.com/sorrentum/sorrentum/issues/568#issuecomment-1724537542

so couple nits and ready IMO

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

File: docs/Contributor_Scoring.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+## Scoring process
+
+### General
+
+We want to evaluate and provide feedback to our team members on different
+aspects of their work.
+
+We don't take non-perfect scores personally but just as a way to understand what
+to improve.
+
```
[the remark this answers is not in the document]
Comment (DanilYachmenev) [reply docs/Contributor_Scoring.md]: no need for notification for each comment, just resolve conversation when done

File: dev_scripts/convert_docx_to_markdown.py [hunk 2]
```diff
@@ -61,7 +61,8 @@ def _convert_docx_to_markdown(
     hsystem.system(remove_figs_folder_cmd)
     # Convert from Docx to Markdown.
     convert_docx_to_markdown_cmd = f"pandoc --extract-media {md_file_figs} -f docx -t markdown_strict -o {md_file} {docx_file}"
-    docker_cmd = f"docker run --rm -it --workdir {work_dir} --mount {mount} {docker_container_name} {convert_docx_to_markdown_cmd}"
+    # Run docker under current uid and gid.
+    docker_cmd = f"docker run --rm --user $(id -u):$(id -g) -it --workdir {work_dir} --mount {mount} {docker_container_name} {convert_docx_to_markdown_cmd}"
```
[the remark this answers is not in the document]
Comment (samarth9008) [reply dev_scripts/convert_docx_to_markdown.py:65]: LGTM. Nice work.

File: docs/Contributor_Scoring.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+- The scoring template is here
+[<span class="underline">Scoring template</span>](https://docs.google.com/spreadsheets/u/2/d/1WsWT8By2hr1VqB6ulIXf3_Rfa0zE2KHI/edit?usp=drive_web&ouid=106425005676808098789&rtpof=true) (this is an Excel spreadsheet since you need to upload it and it needs to be a file and not a Google Sheet).
+
+- Each metric is scored between 1 (poor), 3 (average) and 5 (excellent)
+
+  - We consider 4 as acceptable, anything less than 4 as problematic and needs improve
+
+- We want to score everyone we work with:
+
+  - Initially only people that we supervise, later any anyone
```
Comment (samarth9008) [inline docs/Contributor_Scoring.md:35]: `later any anyone` -> `later on anyone`

**Review by samarth9008 — APPROVED** [review]
Some nits and then its a merge. G!

**Review by gpsaggese — COMMENTED** [review]
LGTM

**Comment by DanilYachmenev** [comment]
integration PR https://github.com/cryptokaizen/cmamp/pull/5510

**Pull request merged**